tailwind.config.js

```diff
@@ -46,6 +46,14 @@ module.exports = {
       lg: '1024px',
       xl: '1280px',
       '2xl': '1536px',
+    },
+    extend: {
+      colors: {
+        raTheme: {
+          scrollthumb: 'var(--link-color)',
```

Comment:
Something more muted would be preferable, as custom styled scrollbars might be used more in the future? Link colors are usually very bright; maybe just the text color?

Reply:
I agree that `--text-color` is a better fit here. Fixed.